processing_chain/src/test/groovy/org/orbisgis/processingchain/ChainProcessMainTest.groovy

```diff
@@ -1,27 +1,45 @@
 package org.orbisgis.processingchain
 
+import org.junit.jupiter.api.Test
 import org.orbisgis.datamanager.JdbcDataSource
+import org.orbisgis.datamanager.h2gis.H2GIS
 import org.orbisgis.processmanagerapi.IProcess
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertEquals
-import static org.junit.jupiter.api.Assertions.assertTrue
-import static org.junit.jupiter.api.Assertions.assertTrue
-import static org.junit.jupiter.api.Assertions.assertTrue
 import static org.junit.jupiter.api.Assertions.assertTrue
 
 
 class ChainProcessMainTest {
 
     public static Logger logger = LoggerFactory.getLogger(ChainProcessMainTest.class)
 
+
+    @Test
+    void OSMGeoIndicators() {
```

Comment:
Is this test necessary? Because it only executes the process

Reply:
In my opinion, it is useful if we make some modifications in the code and we want to debug the geoclimateChain.

Reply:
Sure but this test is already checked in the script ProcessingChainOSMTest and ProcessingChainBDTopoTest. So I guess it's redundant here

Reply:
not exactly the same since it is new to include the LCZ calculation at the end of the geoindicators.

Reply:
No I have added this part of process within the geoClimateChain called "OSMGeoIndicators".

Reply:
My fault ok